include/hpp/core/problem-solver.hh

```diff
@@ -48,6 +48,9 @@ namespace hpp {
       ConfigurationShooterBuilder_t;
     typedef boost::function <SteeringMethodPtr_t (const ProblemPtr_t&) >
       SteeringMethodBuilder_t;
+		typedef std::vector<boost::shared_ptr<model::CollisionObject> > 
+			AffordanceObjectVector_t;
```

Comment:
For homogeneity with previous code, I would rename AffordanceObjectVector_t as AffordanceObjects_t (unless there are several type of containers : vector, list,...)